cmd/internal/cmd/flow/flow.go

```diff
@@ -114,7 +116,10 @@ type node struct {
 }
 
 func runFlows(p *printer.Printer) error {
-	ts := time.Now().UTC().Add(-1 * opts.since)
+	until := time.Now().UTC().Add(-opts.until)
+	since := until.UTC().Add(-opts.since)
```

Comment:
I don't understand why `since` is based on `until` here and not `time.Now()` instead. When I run `fake flow --since 1h --until 30m` I get:
```
now	2023-04-14 14:00:00 +0000 UTC
since	2023-04-14 12:30:00 +0000 UTC
until	2023-04-14 13:30:00 +0000 UTC
```

after


```suggestion
	now := time.Now().UTC()
	until := now.Add(-opts.until)
	since := now.Add(-opts.since)
```

I get the expected

```
now	2023-04-14 14:00:00 +0000 UTC
since	2023-04-14 13:00:00 +0000 UTC
until	2023-04-14 13:30:00 +0000 UTC
```

Reply:
Ah your right, I messed this up.